protostar-rust/src/main.rs

```diff
@@ -29,17 +29,10 @@ fn main_execution() -> Result<()> {
     let scarb_metadata = MetadataCommand::new().inherit_stderr().exec()?;
 
     for package in &scarb_metadata.workspace.members {
-        let protostar_config =
-            rust_test_runner::protostar_config_for_package(&scarb_metadata, package)?;
         let (base_path, dependencies) =
             rust_test_runner::dependencies_for_package(&scarb_metadata, package)?;
```

Comment:
Where are these dependencies resolved now if they got removed from here? Does outputs of `scarb build` handled in the cairo runner account for these?

Reply:
Can you specify what you mean exactly? 
This is not removed:
```
let (base_path, dependencies) =
            rust_test_runner::dependencies_for_package(&scarb_metadata, package)?;
```
This is removed:
```
let protostar_config =
            rust_test_runner::protostar_config_for_package(&scarb_metadata, package)?;
```
as we agreed and it's not done anywhere instead, we agreed to resign from this in https://github.com/software-mansion/protostar/pull/2059#discussion_r1226632012

Reply:
I think we meant to remove the config for serialization, not the config in general. ProtostarTestConfig is used by the test runner, we cannot remove it

Reply:
@cptartur can u confirm/clarify?

Reply:
Discussed offline, @piotmag769 misunderstood, @cptartur pls either clarify or resolve, thx.

Reply:
Those are resolved by scarb under the hood when we call build, does it clarifies it @cptartur ?

Reply:
Yes